gui-agent/vmside.c

```diff
@@ -1052,11 +1058,18 @@ static void process_xevent_selection(Ghandles * g, XSelectionEvent * ev)
                 g->utf8_string_atom, g->qprop,
                 g->stub_win, ev->time);
     else
-        send_clipboard_data(g->vchan, g->stub_win, (char *) data, len);
-    /* even if the clipboard owner does not support UTF8 and we requested
-       XA_STRING, it is fine - ascii is legal UTF8 */
+        if (type == XInternAtom(g->display, "INCR", False)) {
+            char INCR_WARNING[] =
+                "Qube clipboard size over 256KiB and X11 INCR protocol support is not implemented!\n";
+            send_clipboard_data(g->vchan, g->stub_win, (char *) &INCR_WARNING,
+                                sizeof(INCR_WARNING), g->protocol_version);
```

Comment:
Test says there is extra `\x00` copied:
```suggestion
                                sizeof(INCR_WARNING)-1, g->protocol_version);
```

Reply:
I will do it.

One other possibility that comes to mind would be sending `char something[MAX_CLIPBOARD_BUFFER_SIZE+1] = {0}` to trigger the xside oversized detection mechanism. A little bit wasteful but it will work.